hijack/urls.py

```diff
@@ -1,9 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.db import models
 from django.urls import path, re_path
 
 from hijack import settings as hijack_settings, views
 
+
+def _get_user_url_pattern(pk=None, pattern=None):
+    pk = pk or get_user_model()._meta.pk
+    pattern = pattern or hijack_settings.HIJACK_USER_URL_PATTERN
+
+    if pattern:
+        return re_path(pattern, views.acquire_user_view, name="acquire")
+    elif isinstance(pk, (models.IntegerField, models.AutoField)):
+        return path("acquire/<int:pk>/", views.acquire_user_view, name="acquire")
+    elif isinstance(pk, models.UUIDField):
+        return path("acquire/<uuid:pk>/", views.acquire_user_view, name="acquire")
+    elif isinstance(pk, models.SlugField):
+        return path("acquire/<slug:pk>/", views.acquire_user_view, name="acquire")
```

Comment:
```suggestion
    path_args = (views.acquire_user_view, name="acquire")
    if pattern:
        return re_path(pattern, *path_args)
    elif isinstance(pk, (models.IntegerField, models.AutoField)):
        return path("acquire/<int:pk>/", *path_args)
    elif isinstance(pk, models.UUIDField):
        return path("acquire/<uuid:pk>/", *path_args)
    elif isinstance(pk, models.SlugField):
        return path("acquire/<slug:pk>/", *path_args)
```

Reply:
Hm... I don't really mind the repetition here. It doesn't really save any lines (but adds one) and also doesn't add to readability.